docs/content/1.getting-started/2.migration.md

```diff
@@ -367,6 +367,59 @@ In addition to the renamed components, there are lots of changes to the componen
 This change affects the following components: `Breadcrumb`, `HorizontalNavigation`, `InputMenu`, `RadioGroup`, `Select`, `SelectMenu`, `VerticalNavigation`.
 ::
 
+2. `UNotifications` component is no longer available and needed due to the usage of `UApp`:
+
+```diff
+<template>
+- <UNotifications />
+</template>
+```
+
+3. `USlideover` now accepts `v-model:open` instead of `v-model`. Also, the content inside needs to be wrapped inside the `template #body`:
+
+```diff
+<template>
+- <USlideover v-model="isCartOpen">
++ <USlideover v-model:open="isCartOpen">
+
+- <USlideover v-model="isCartOpen">
+-   <CartOrderSummary />
+- </USlideover>
++ <USlideover
++   v-model:open="isCartOpen"
++   title="Order Summary"
++ >
++   <template #body>
++     <CartOrderSummary />
++   </template>
++ </USlideover> 
+</template>
+```
+
+4. `UVerticalNavigation` is now `UNavigationMenu`:
```

Comment:
This is already explained in the Renamed components section.

Reply:
Ok, removing